app/AppKernel.php

```diff
@@ -63,5 +67,9 @@ public function getLogDir()
     public function registerContainerConfiguration(LoaderInterface $loader)
     {
         $loader->load($this->getRootDir().'/config/config_'.$this->getEnvironment().'.yml');
+
+        if ('test' !== $this->getEnvironment()) {
+            $loader->load($this->getRootDir().'/config/geocoder.yml');
```

Comment:
This will be soon be loaded in every environment as I'm now providing a custom `DummyGeocoder` mock implementation for testing.